### ISSUE TITLE
Removing app from Kaffeine

### ISSUE THREAD

**Issue by guibgoulart** [romainbutteaud/Kaffeine#41]
Hello! First of all, congratulations for this amazing tool.
I'd like to know how can i know *for sure* that Kaffeine is no longer pinging my app? I'd like to edit the sleep time and be certain that when i wish it no longer pings it.

Thanks!

**Comment by RishiHQ** [comment]
Hi, thank you for this tool, it was quite handy to save us some money in upgrading Heroku for a while. But now we are ready to upgrade.

So, we're interested in making sure our app is removed from Kaffeine. 

I hit the `remove your app?` button, which I wasn't sure if it worked. I don't see any network requests in the Chrome Network tab when I click it, so it doesn't seem to work?

Would love any help, thank you!

**Comment by RishiHQ** [comment]
Ah, another solution had the solution: We can pop over to: http://kaffeine.herokuapp.com/#decaf

**Comment by abdulloooh** [comment]
Oh thanks for this

**Comment by kinamod** [comment]
So the problem is that the link on the page doesn't seem to launch the modal for some reason...
If you follow the link above then it works, but if you click the "remove you app?" link, it doesn't seem to do anything
![image](https://user-images.githubusercontent.com/3469955/103298924-4cd77000-49f3-11eb-9a90-cd9cbbb92e69.png)

**Comment by abdulloooh** [comment]
Exactly, so just use this https://github.com/romainbutteaud/Kaffeine/issues/41#issuecomment-666010014

**Comment by gyz15** [comment]
I have tried these way to open it, just choose either one that suits you:
1. Follow the [comment by RishiHQ](https://github.com/romainbutteaud/Kaffeine/issues/41#issuecomment-666010014)
2. Using right click the _'remove your app?'_ link and open the link in new tab.
3. Ctrl + click the _'remove your app?'_ link (Same with solution 2 but faster).
